gen/generator.jl

```diff
@@ -39,7 +51,7 @@ for target in JLLEnvs.JLL_ENV_TRIPLES
         push!(args, "-DSUN64 -DLONGBLAS='long long'")
     end
 
-    header_files = [cholmod_h, cholmod_blas_h, SuiteSparseQR_C_h, umfpack_h]
+    header_files = [cholmod_h, cholmod_blas_h, SuiteSparseQR_C_h, umfpack_h, klu_h]
```

Comment:
Is it intended to not include `amd_h`, `colamd_h` and `ccolamd_h`? Perhaps, `klu_h` includes these headers indirectly?

Reply:
Unintentional, let's see what happens